src/pages/index.tsx

```diff
@@ -4,24 +4,40 @@ import AboutUs from "../components/aboutUs/AboutUs";
 import Services from "../components/services/Services";
 import Contacts from "../components/contacts/Contacts";
 import Works from "../components/works/Works";
+import Divider from "../components/divider/divider";
+import { useEffect, useState } from "react";
 
+function useWindowSize() {
```

Comment:
do not use hooks like this in your case. use responsive images tecnhique instead https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images